environments/staging.yaml

```diff
@@ -2,4 +2,5 @@ continuumLogin:
   url: "https://continuumtest--cdn-journal.elifesciences.org/submit"
 
 continuumAuth:
-  port: 3001
\ No newline at end of file
+  port: 3001
+  continuumApiUrl: "https://continuumtest--gateway.elifesciences.org"
```

Comment:
`http://continuumtest--gateway.elife.internal` one doesn't work?
```
$ kubectl run --rm --stdin --generator=run-pod/v1 --restart=Never --image=busybox sh
If you don't see a command prompt, try pressing enter.
wget http://continuumtest--gateway.elife.internal/ping
Connecting to continuumtest--gateway.elife.internal (10.0.2.81:80)
saving to 'ping'
ping                 100% |********************************|     4  0:00:00 ETA
'ping' saved
pod "sh" deleted
```

Reply:
i'll try it again after the demo. it didn't work when we tried it but it might have been due to a misconfiguration somewhere else we fixed after